copier.yaml

```diff
@@ -170,3 +170,19 @@ default_py:
       value: '3.12'
   help: 'Choose the default Python version for development, documentation generation, and package build:'
   type: str
+readme_content:
+  default: |-
+    [% if project_name == "Serious Scaffold Python" -%]
+      [% include 'includes/sample/readme_content.jinja' %]
+    [%- else -%]
+    ## title 1
+
+      content 1
+
+    ## title 2
+
+      content 2
```

Comment:
Usually, developers maybe are not ready to fill the readme content, how about considering to leave it blank.🤔

Reply:
Or just add an annotation here or give a choice to leave a blank, for example:
```md
# Fill the readme content here
```

Reply:
That is totally OK, actually I was thinking how to demostrate a multiline string here.

Reply:
But it's ok and althogh the readme content placeholder seams to have too many text description.

Reply:
> That is totally OK, actually I was thinking how to demostrate a multiline string here.

Ah,I just consider sometimes we want to skip everything for quick start.

Reply:
PTAL, let me know if there is any misunderstand for me.